apps/i18n/common/en_us.json

```diff
@@ -1658,6 +1659,7 @@
   "print": "Print",
   "privacyPolicy": "Privacy Policy",
   "privacyPracticesForChildren": "See our privacy practices for children",
+  "professionalLearningImageDescription": "A focused facilitator wearing a Code.org name tag is working at a table during a workshop.",
```

Comment:
Love the 'focused' descriptor here

Reply:
We use active voice elsewhere: "works" instead of "is working"